package.json

```diff
@@ -48,8 +46,9 @@
           "modules": false,
           "targets": {
             "browsers": [
-              "last 2 versions",
-              "ie >= 10"
+              ">0.25% in GB",
```

Comment:
See https://jamie.build/last-2-versions 

(Although not quite as aggressive as that suggestion in the article, still supports IE 10+)